laalaa/settings/base.py

```diff
@@ -25,9 +25,10 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = os.environ.get('SECRET_KEY', 'DEV_KEY')
 
-DEBUG = False
+#MUST CHANGE THESE BACK BEFORE COMMITTING
+DEBUG = True
 
-TEMPLATE_DEBUG = False
+TEMPLATE_DEBUG = True
 
```

Comment:
need to create a local.py in the same dir as settings and you can override settings in this